lambdatimes/src/components/Content/Cards.js

```diff
@@ -1,15 +1,22 @@
 import React, { Component } from 'react';
 import Card from './Card';
+import PropTypes from "prop-types";
 
 const Cards = props => {
   return (
     <div className="cards-container">
       {/* Using the cards prop, map over the list creating a 
-          new Card component for each passing the card as the only prop*/}
+          new Card component for each passing the card as the only prop*/
+          props.cards.map(card => {
+            return(<Card key={Date.now() * Math.random()} card = {card} />)
+          })}    
```

Comment:
not sure if the use of two methods in the key is worth it to generate a random unique key but i like the inventiveness, it would be easier to grab maybe the headline from the card